http/server.go

```diff
@@ -73,6 +73,16 @@ type acceptReq struct {
 	QuoteHash string
 }
 
+type acceptRes struct {
+	Signature                 string `json:"signature"`
+	BitcoinDepositAddressHash string `json:"bitcoinDepositAddressHash"`
```

Comment:
why would we send this bitcoindepositaddresshash as part of the response? the user should have the address already in the quote. AFAIU the only necessary response value is the Signature.